core/src/main/scala/kafka/server/KafkaApis.scala

```diff
@@ -993,6 +1000,15 @@ class KafkaApis(val requestChannel: RequestChannel,
     }
   }
 
+  def handleApiVersionRequest(request: RequestChannel.Request) {
+    val responseHeader = new ResponseHeader(request.header.correlationId)
+    val responseBody = if (!authorize(request.session, Describe, Resource.ClusterResource))
+      ApiVersionResponse.fromError(Errors.CLUSTER_AUTHORIZATION_FAILED)
```

Comment:
Is this right? Doesn't this mean that cluster Describe access needs to be given to all clients?